atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -366,7 +366,7 @@ def stream_filesystem(self, task_id, filesystem_regex):
 
     def run(self):
         base_image = self.workflow.builder.base_image
-        if base_image.namespace != 'koji' or base_image.repo != 'image-build':
+        if not self.workflow.builder.custom_base_image:
```

Comment:
I'd expect this to be
```
if not (self.workflow.builder.custom_base_image or self.workflow.builder.base_from_scratch):
```

Reply:
I can make the change, but we're currently only running pre_add_filesystem for custom base image builds, not scratch builds.  And from what rcerven said when I asked about setting self.parent_images[] in InsideBuilder, I thought that scratch didn't get add_filesystem.

Reply:
this is fine for now, I've created new story to actually support custom base image with multistage, as they aren't working now

Reply:
Do you mean keep with the change I've proposed or change to Tim's suggestion?

Reply:
keep what you have